text/0000-run-image-sbom.md

```diff
@@ -0,0 +1,72 @@
+# Meta
+[meta]: #meta
+- Name: Run Image Structured SBOM
+- Start Date: 2021-10-25
+- Author(s): aemengo
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes the following -
+
+- A run image can contain a layer holding an SBOM (in format of CycloneDX or SPDX or Syft) of installed packages at `/cnb/sbom/bom.<ext>.json`, where `<ext>` will be `cdx` for CycloneDX documents, `spdx` for SPDX documents, and `syft` for Syft documents. These will initially be the only 3 supported SBOM types.
+- The digest of this layer is written as a `LABEL` on the corresponding run image with the key: `io.buildpacks.base.sbom`.
+- For the sake of parity, the digest of buildpack SBOM layers (created by lifecycle) is written as a `LABEL` on the corresponding run image with the key: `io.buildpacks.app.sbom`.
+
```

Comment:
In the future, we may want to add a flag to lifecycle to include build-time SBoMs in the app image. We could use labels like `io.buildpacks.build.base.sbom` and `io.buildpacks.build.app.sbom`.

I **don't** think this means we should rename `io.buildpacks.base.sbom` to `io.buildpacks.launch.base.sbom` -- just thought it might be worth pointing this out.

Reply:
Yes, we _may_. I personally find it hard to base decisions purely on what may happen. I'd be curious if other explicitly have this use-case.